### ISSUE TITLE
README.ms instructions build failing

### ISSUE THREAD

**Issue by sebastianconcept** [mpw/Objective-Smalltalk#13]
Hi @mpw , I'd like to build the stsh scheme it but I'm stuck with this error

`MPWFoundation/MPWFoundation.h' file not found`

Do I have to manually add it somehow?

I did try to build MPWFoundation in a sibling dir, it did built with some warnings.

But then on Objective-Smalltalk I have that file not found issue. 

Is a patch issue? What I am missing?

XCode 13.4.1

**Comment by mpw** [comment]
Did you follow step 3 in the README?

3. Open Objective-Smalltalk/objective-smalltalk.xcworkspace

**Comment by sebastianconcept** [comment]
Ah, I've overlooked and opened `ObjectiveSmalltalk.xcodeproj` instead.

I see `objective-smalltalk.xcworkspace` build both projects.

Now when I build `objective-smalltalk.xcworkspace` goes well until this 1 error:

'ObjectiveSSH/SSHConnection.h' file not found

**Comment by mpw** [comment]
Hmm...I may have been a bit too clever trying to work around the difficulty of building libssh fat.

In MPWFoundation, build the ObjectiveSSH target once manually.  That should put the headers where they are needed.

**Comment by sebastianconcept** [comment]
Thanks for the follow up Marcel.

When doing that, it complain about lack of team for signing the build. I've set my personal team and when retrying the build, I've got:

'libssh/libssh.h' file not found

<img width="885" alt="Screen Shot 2022-08-13 at 18 16 14" src="https://user-images.githubusercontent.com/475508/184510934-d9c99a97-1197-4ec0-bbff-32ce683c65f4.png">

**Comment by sebastianconcept** [comment]
Seems to expect that lib to be in `/usr/local/Cellar/libssh/...` and I don't have it there.

Is a version of SSH you installed with brew?

**Comment by mpw** [comment]
Yep, brew it was.  I keep going back-and-forth on how to integrate external dependencies like that.  Quickest/easiest solution is to add your location to the search paths.

**Comment by mpw** [comment]
OK, I've removed the dependency on ObjectiveSSH, by introducing an SSHConnection protocol in MPWFoundation.

That means you should now be able to compile Stsh.framework (and thus stsh) without needing libssh.

**Comment by sebastianconcept** [comment]
Thanks @mpw that seems to have helped a lot because the build now goes a long way.

Now I'm stuck with:
`error: Build input file cannot be found: '/Users/seb/git/Objective-Smalltalk/stsh/stsh-Prefix.pch' (in target 'stsh' from project 'ObjectiveSmalltalk')`

**Comment by mpw** [comment]
That looks a lot like a generic Xcode build system bug/problem. Typical Xcode incantations should help, starting with cleaning the target, maybe restart Xcode, the works... :-/

**Comment by sebastianconcept** [comment]
I've thought that. I did that cleaning and rebuilt but got the same.
It could be the version because I've installed an older one for a specific iOS version that I've targeted for some reason. Not important now. I can upgrade.

What XCode version are you using?

**Comment by mpw** [comment]
13.4.

Hmm.

**Comment by mpw** [comment]
I've disabled the stsh prefix header in the Build settings of the stsh target.
